app/components/ak-tooltip/index.hbs

```diff
@@ -6,38 +6,30 @@
   {{on 'mouseenter' this.handleShowTooltip}}
   {{on 'mouseleave' this.handleHideTooltip}}
 >
-
   {{yield}}
 
   {{#unless @disabled}}
-    {{#if this.anchorRef}}
-      <EmberWormhole @to='ak-component-wormhole'>
-        <div role='tooltip' local-class='ak-tooltip-container-root'>
-          <AkPopover
-            data-test-ak-tooltip-popover
-            @anchorRef={{this.anchorRef}}
-            @placement={{@placement}}
-            @arrow={{@arrow}}
-            @arrowColor={{or @color 'dark'}}
-            @modifiers={{this.modifiers}}
-            local-class='{{if @arrow "ak-tooltip-arrow-popover-root"}}'
-          >
-            <div
-              data-test-ak-tooltip-content
-              local-class='ak-tooltip-root ak-tooltip-color-{{or
-                @color
-                "dark"
-              }}'
-            >
-              {{#if (has-block 'tooltipContent')}}
-                {{yield to='tooltipContent'}}
-              {{else}}
-                {{@title}}
-              {{/if}}
-            </div>
-          </AkPopover>
-        </div>
-      </EmberWormhole>
-    {{/if}}
+    <AkPopover
+      data-test-ak-tooltip-popover
+      role='tooltip'
+      local-class='ak-tooltip-container-root'
+      @anchorRef={{this.anchorRef}}
```

Comment:
I had a use for the `renderInPlace` prop (for the tooltip specifically) during the project analysis settings revamp. I think it'd be nice to have that functionality here also. It's not a blocker for this PR though.

Reply:
done